server/fleet/agent_options.go

```diff
@@ -8,7 +8,8 @@ type AgentOptions struct {
 	// Config is the base config options.
 	Config json.RawMessage `json:"config"`
 	// Overrides includes any platform-based overrides.
-	Overrides AgentOptionsOverrides `json:"overrides,omitempty"`
+	Overrides               AgentOptionsOverrides `json:"overrides,omitempty"`
+	CommandLineStartUpFlags json.RawMessage       `json:"command_line_flags"`
```

Comment:
`command_line_flags` (add "startup" somewhere?)

Or just make the distinction between the other flags in the documentation. I always get confused as to which is CLI and which isn't.